tmail-backend/tmail-third-party/openpaas/src/test/resources/docker-openpaas-setup.yml

```diff
@@ -84,3 +117,9 @@ services:
       - OPENPASS_BASIC_AUTH=YWRtaW5Ab3Blbi1wYWFzLm9yZzpzZWNyZXQ=
     networks:
       - emaily
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "-u", "admin&admin@open-paas.org:secret123", "http://esn_sabre:80","-X", "PROPFIND" ]
+      start_period: 40s
+      interval: 10s
+      timeout: 10s
+      retries: 10
```

Comment:
we can modify the `depends_on` logic for increased effectiveness.
eg:
old
```yaml
    depends_on:
      elasticsearch
```

new:
```yaml
    depends_on:
      elasticsearch:
        condition: service_healthy
```